editor/src/components/canvas/canvas-strategies/strategies/set-flex-gap-strategy.tsx

```diff
@@ -125,7 +125,10 @@ export const setFlexGapStrategy: CanvasStrategyFactory = (
       resizeControl,
       controlWithProps({
         control: FloatingIndicator,
-        props: { ...props, color: colorTheme.gapControls.value },
```

Comment:
Does this mean that `gapControls` should be updated?

Reply:
mmmh it's still there for the striped background, maybe worth renaming it to `gapControlsBg` or something like that?